doc/dev/quick_guide.rst

```diff
@@ -25,6 +25,34 @@ Example:
    ./install-deps.sh
    ./do_cmake.sh -DWITH_MANPAGE=OFF -DWITH_BABELTRACE=OFF -DWITH_MGR_DASHBOARD_FRONTEND=OFF
 
+Introducing new dependencies
+----------------------------
+Adding external dependencies should be avoided whenever possible, in order to
+reduce maintenance overhead. In case the new dependency is absolutely essential
+and must be introduced, please verify that its package is available in the
+repositories of all the supported distributions. In addition, pay attention to
+its version, since sometimes versions do not match across distributions.
+
+For RHEL and CentOS you can check if a dependency is available with:
+
+.. prompt:: bash $
+
+   dnf search <package_name>
+
+Make sure you add Extra Packages for Enterprise Linux (EPEL) repository to your
+``/etc/yum.repos.d/``.
+Instructions for doing so are found on the Fedora Wiki:
+https://fedoraproject.org/wiki/EPEL
```

Comment:
not sure why you single out EPEL. but for the sake of completeness, the same applies to CodeReady Linux Builder / PowerTools, and probably need to enable some modules required for building Ceph, like `javapackages-tools` for building cephfs's java JNI binding. some PPA repo for debian derivatives. `install-deps.sh` is designed to hide these kind of details.

Reply:
I *think* `dnf` is not available before RHEL/CentOS 8, but I could be wrong.